x/incentives/spec/02_state.md

```diff
@@ -0,0 +1,58 @@
+<!--
+order: 2
+-->
+
+# State
+
+## Incentives management
+
+All the incentives that is going to be provided are locked into `IncentivePool` until it's released to the users after a specific period of time.
```

Comment:
Where is the set of users that constitute the pot stored?

Reply:
I am not sure where we use those set of users.

Reply:
We have to use them upon releasing the funds from the pot, no?

Reply:
That's gotten from the querying of the lockup module.